deployment/gke/gke-kube-dashboard-rbac.yml

```diff
@@ -1,12 +1,13 @@
-apiVersion: rbac.authorization.k8s.io/v1beta1
+# When enable RBAC on GKE, Dashboard is not accessible without bellow CRB
```

Comment:
This file is specific to GKE, do we really need to put it this project?
Peoples that have activated RBAC on their clusters should already have this issue without using the workflow-controller.

Maybe we can add an option in the Helm chart to activate or deactivate the creation of RBAC resources. What do you think? Can be done in another PR.

Reply:
Agree. Will remove it for now. Maybe will open a separate PR with GKE related files